Vungle/VungleRouter.m

```diff
@@ -205,16 +208,22 @@ - (void)requestRewardedVideoAdWithCustomEventInfo:(NSDictionary *)info delegate:
 - (void)requestBannerAdWithCustomEventInfo:(NSDictionary *)info size:(CGSize)size delegate:(id<VungleRouterDelegate>)delegate {
     [self collectConsentStatusFromMoPub];
 
-    // Verify if PlacementID is nil (first MREC request) or PlacementID is the same one requested
-    if (self.bannerPlacementID != nil && ![[info objectForKey:kVunglePlacementIdKey] isEqualToString:self.bannerPlacementID]) {
-
-        MPLogInfo(@"A banner ad type has been already instanciated. Multiple banner ads are not supported with Vungle iOS SDK version %@ and adapter version %@.", VungleSDKVersion, VungleAdapterVersion);
+    // Verify if PlacementID is nil (first MREC or banner request) or PlacementID is the same one requested
+    if (self.mrecPlacementID != nil && ![[info objectForKey:kVunglePlacementIdKey] isEqualToString:self.mrecPlacementID]) {
+        MPLogInfo(@"A MREC ad type has been already instantiated. Multiple MREC ads are not supported with Vungle iOS SDK version %@ and adapter version %@.", VungleSDKVersion, VungleAdapterVersion);
+        [delegate vungleAdDidFailToLoad:nil];
+    } else if (self.bannerPlacementID != nil && ![[info objectForKey:kVunglePlacementIdKey] isEqualToString:self.bannerPlacementID]) {
```

Comment:
As I wrote in AdMob side, we might want to set `nil` for `bannerPlacementID` and `mrecPlacementID` after finishing displaying banner/mrec ad, so `router` class can allow to load for next banner/mrec ad with "different" placement ID.

Reply:
@ashinagawa Hi Aki, according to your comments, I set nil for bannerPlacementID and mrecPlacementID after finishing displaying banner/mrec ad for both MoPub and AdMob Adapter, please review the changes in these two PRs. Thanks a lot. :)

Reply:
@YueVungle Please verify "refresh" is working correctly after your update.

Reply:
@ashinagawa Hi Aki, I have verified that "refresh" works correctly after the update here. :)
It seems like this PR has passed the code review, may I close this RR :)